ocs_ci/ocs/ui/views.py

```
@@ -1126,6 +1126,70 @@
 
+# Fusion Access for SAN — Storage Cluster creation locators
+FUSION_ACCESS_STORAGE_CLUSTER_LOCATORS = {
+    # Left nav: Storage > Fusion Access for SAN
+    "fusion_access_for_san_nav": (
+        "//a[normalize-space()='Fusion Access for SAN']",
+        By.XPATH,
+    ),
+    # "Create storage cluster" button on the empty-state page (no cluster yet)
+    "create_storage_cluster_btn": (
+        "//button[normalize-space()='Create storage cluster']"
+        " | //button[contains(@data-test,'create-storage-cluster')]",
```

Ruff (0.16.0)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)